packages/framework/src/Component/Domain/Multidomain/MultiDomainEntityClassProviderInterface.php

```diff
@@ -2,15 +2,28 @@
 
 namespace Shopsys\FrameworkBundle\Component\Domain\Multidomain;
 
+/**
+ * Interface MultiDomainEntityClassProviderInterface
+ *
+ * Serves as provider for ignored and manual entities.
+ *
+ * MultidomainEntityClassFinder finds entities that has identifier called $domainId, but sometimes this assumption
+ * is not implemented for some reason. This interface adds option to ignore or manually add entities that does not match
+ * this assumption.
+ */
 interface MultiDomainEntityClassProviderInterface
 {
 
     /**
+     * Return entities FQNs that has identifier called $domainId but is not multi domain entity.
```

Comment:
"...that have identifier...but are not multidomain entities" :slightly_smiling_face: I noticed that you use the term "multi domain" as two words but it should be just one word - "multidomain". Eg. existing `MultidomainEntityClassFinder` vs your new `MultiDomainEntityClassProviderInterface` - please revise it.